src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Options/LocalClientSettingsStorageLocation.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Microsoft.CodeAnalysis.Options;
+
+/// <summary>
+/// Specifies that the option should be stored into local client settings storage.
+/// </summary>
+/// <remarks>
+/// Unlike LocalUserRegistryOptionPersister, which accesses the registry directly this storage is managed by VS Settings component.
```

Comment:
cref? But thanks for clarifying this is not the registry one. (Although does the registry one now redirect back to the store anyways?)

Reply:
cref does not work due to code style layering :(

Reply:
Oh right boo.

Reply:
Yeah, this should really not be in this layer (see TODO comment) but for now it has to.